### ISSUE TITLE
Make discardThreshold configurable

### ISSUE THREAD

**Issue by farnoy** [Rob--W/bookmark-container-tab#14]
I noticed that the discard threshold when opening folders of bookmarks is hardcoded. Would it be possible to add an option to increase it?

**Comment by Rob--W** [comment]
Yes that's possible. I started with a hard-coded value to see if it would be good enough. In the original issue that resulted in this feature (#9), someone actually asked to lower the threshold to 0. There's clearly no one-size-fits-all, so I'm favorable towards adding options to customize the setting.

The required code change is quite simple, and would be similar to https://github.com/Rob--W/dont-track-me-google/commit/d7e3d491ac990a0cc39575968d2069f8c7177c15#diff-a2661882d2a4772ee7877c2794c66d81

One thing is that the extension is localized, and that the new options page should ideally be localized as well.

**Comment by farnoy** [comment]
Thanks, I'll take a stab at it soon.
